pkg/prompts/prompts.go

```diff
@@ -280,11 +280,11 @@ func (*realPrompter) CaptureID(promptStr string) (ids.ID, error) {
 		Validate: validateID,
 	}
 
-	IDStr, err := prompt.Run()
+	idStr, err := prompt.Run()
```

Comment:
Huh, why does the linter not complain here? We always use `ID` for such labels?

Reply:
It's ok with "id" and "ID", but it doesn't like "Id" and it doesn't like local scoped things starting with capital letters:

```
pkg/prompts/prompts.go:283:2: unexported-naming: the symbol IDStr is local, its name should start with a lowercase letter (revive)
```